modules/virt-fips-support.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+[id="virt-fips-support_{context}"]
+= Support for FIPS cryptography
+
+If FIPS mode is xref:../../installing/installing-fips.adoc#installing-fips[set up correctly at the OpenShift level], it is automatically enabled and no additional setup is needed for {VirtProductName}. FIPS must be enabled before your cluster's operating system boots for the first time.
```

Comment:
s/OpenShift/{product-title}

I think @ousleyp should weigh in here, but I think the best place for this note would be higher up in the assembly as part of the list of requirements that your OCP cluster must meet, and not as a separate module.

You could word the list item something like this: "Your cluster is deployed with FIPS mode enabled. xref:../../installing/installing-fips.adoc#installing-fips[Support for FIPS cryptography] must be set up before the operating system that your cluster uses boots for the first time."

The second sentence in my suggestion could also be an admonition below the list item.

Reply:
My interpretation of this PR is that FIPS isn't a requirement itself, but it's something that OpenShift Virtualization now supports, as long as you set it up properly. Is that correct, @sabrinajess? 

Regardless, I agree with @sjhala-ccs that this should be placed higher up in the assembly. Because similar content in this assembly is added directly to the assembly text, this content should probably follow suit.

Reply:
Also, you can't put an xref in a module, which is also why the other content in this assembly has been added directly.

Reply:
@ousleyp Is correct that it's not a requirement or anything that needs to be done, but added support. I'll update the assembly with this feedback.